src/app/speechtotext/speechtotext.component.spec.ts

```diff
@@ -31,4 +47,89 @@ describe('SpeechtotextComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should create div with id spch', () => {
+    let compiled = fixture.debugElement.nativeElement;
+    expect(compiled.querySelector('div#spch')).toBeTruthy();
```

Comment:
You should write tests which tests something meaningful.

Reply:
removed